src/github.com/getlantern/lantern-ui/app/partials/feed.html

```diff
@@ -1,6 +1,6 @@
 <a class="item" target="_blank" href="{{feed.link}}" ng-click="trackFeed(feedsTitle)">
   <div ng-if="feed.image" class="feed-thumbnail">
-    <img ng-src="{{feed.image}}"/>
+    <img ng-src="{{feed.image}}" on-error="hideImage(feed)"/>
```

Comment:
shouldn't this be `onerror` instead of `on-error` @fffw? Or are they equivalent?